app/renderer/components/common/dropdown.js

```diff
@@ -38,6 +39,12 @@ class SettingDropdown extends ImmutableComponent {
   }
 }
 
+class PanelDropdown extends ImmutableComponent {
+  render () {
+    return <FormDropdown data-isPanel='true' {...this.props} />
```

Comment:
you're actually passing a string here, so this is actually truthy and not `true`. For such cases you'd need to pass as a boolean like `data-isPanel={true}`, but standard will complain. Hidden values are evaluated to `true` anyway and make standard happy, so `<FormDropdown data-isPanel {...this.props} />` is the right take.